src/MessagingThreadHistory/MessageMetadata.tsx

```diff
@@ -26,16 +26,25 @@ export const MessageMetadata: React.FC<
     <div ref={ref} className={classNames(cssClass("Message--container"), className)}>
       <div className={cssClass(`Message--${placement}`)}>{children}</div>
       {readStatusText && <div className={cssClass("ReadReceipt")}>{readStatusText}</div>}
-      {errorMsg && formErrorContainer(errorMsg)}
+      {errorMsg && formErrorContainer(errorMsg, placement)}
     </div>
   );
 });
 
-function formErrorContainer(errorMsg: React.ReactNode): JSX.Element {
+function formErrorContainer(
+  errorMsg: React.ReactNode,
+  placement: "left" | "right" | "center" | "fullWidth",
```

Comment:
Do we use this same enum elsewhere? Might be good to make it its own type if so.

Reply:
addressed in this commit: https://github.com/Clever/components/pull/677/commits/aa5609a3fdb5a705d256a9b7e6575e94f0327292